profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.callbacks.inc

```diff
@@ -99,6 +99,13 @@ function nexteuropa_multilingual_language_negotiation_split_suffix($path, $langu
     return array(FALSE, $path);
   }
 
+  // Handle Estonian language enabled and user adding title_field_et to a view.
+  // @link https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-107
+  $parts = explode('/', $path);
+  if (count($parts) > 0 && array_pop($parts) == 'title_field_et') {
```

Comment:
Message for the reviewer: maybe we should include a longer path in the check so that we are sure we are tackling the case at hand.